src/map/Map.js

```diff
@@ -35,11 +35,11 @@ L.Map = L.Evented.extend({
 		zoom: undefined,
 
 		// @option minZoom: Number = undefined
-		// Minimum zoom level of the map. Overrides any `minZoom` option set on map layers.
+		// Minimum zoom level of the map. Any `minZoom` set on map layers overrides this value.

```

Comment:
I think the right thing to do would be to change this into «`minZoom: Number = *`» and «Minimum zoom level of the map. If not specified and at least one `GridLayer` or `TileLayer` is in the map, the lowest of their `minZoom` options will be used instead.»